lib/models/search_record_model.dart

```diff
@@ -1,6 +1,22 @@
-
-//  TODO : Create a hive model with 
+//  TODO : Create a hive model with
 //  the following fields:
 //  - id (int)
 //  - Origin (String)
-//  - Destination (String)
\ No newline at end of file
+//  - Destination (String)
+import 'package:hive_flutter/adapters.dart';
+part 'search_record_model.g.dart';
+
+@HiveType(typeId: 0)
+class HiveModel {
```

Comment:
Change the name HiveModel into SearchRecordModel

Reply:
Done